notebooks/NB_FINAL.ipynb

```diff
@@ -1126,7 +1128,7 @@
    "source": [
     "#### Particionando os Dados\n",
     "\n",
-    "Aqui é onde selecionaremos os **atributos preditore**(variáveis independentess), aqueles que serão utilizados para encontrar nossso atributso-alvo (variável dependente), ou seja, o número total de notificações de incidências de dengue. Particionaremos os dados com a função `train_test_split` da biblioteca sklearn na proporção **80% para treino** e **20% para teste**, respectivamente.\n"
+    "Aqui é onde selecionaremos os **atributos preditores** (variáveis independentes), aqueles que serão utilizados para encontrar nossso atributo-alvo (variável dependente), ou seja, o número total de notificações de incidências de dengue. Particionaremos os dados com a função `train_test_split` da biblioteca sklearn na proporção **80% para treino** e **20% para teste**, respectivamente.\n"
```

Comment:
"o número absoluto de casos notificados."
Change to:
"...ou seja, o número absoluto de casos notificados de dengue para o Estado do Ceará."